code/edge/src/main/java/com/adobe/marketing/mobile/Edge.java

```diff
@@ -46,16 +46,12 @@ public static String extensionVersion() {
 	public static void registerExtension() {
 		MobileCore.registerExtension(
 			EdgeExtension.class,
-			new ExtensionErrorCallback<ExtensionError>() {
-				@Override
-				public void error(ExtensionError extensionError) {
-					MobileCore.log(
-						LoggingMode.ERROR,
-						LOG_TAG,
-						"Edge - There was an error registering the Edge extension: " + extensionError.getErrorName()
-					);
-				}
-			}
+			extensionError ->
+				MobileCore.log(
+					LoggingMode.ERROR,
+					LOG_TAG,
+					"Edge - There was an error registering the Edge extension: " + extensionError.getErrorName()
```

Comment:
Can you change this log message to the new API?

Reply:
in the next PR